app/http.php

```diff
@@ -327,4 +378,50 @@
     }
 });
 
+function getDomains(Database $dbForConsole, &$lastSyncUpdate, &$domains, $pools)
+{
+    go(function () use ($dbForConsole, &$lastSyncUpdate, &$domains, $pools) {
```

Comment:
Hmm, can we try to add `sleep(10)` in here, all over the place? Just to ensure if anything goes wrong here, master doesn't freeze and can still process HTTP requests.